x-pack/plugins/uptime/public/components/monitor/ping_list/columns/ping_timestamp/step_image_popover.tsx

```diff
@@ -52,6 +52,7 @@ export const StepImagePopover: React.FC<StepImagePopoverProps> = ({
       />
     }
     isOpen={isImagePopoverOpen}
+    closePopover={() => {}}
```

Comment:
it closes auto on mouseOut, this is needed to satisfy typescript

Reply:
Might want to include that as a comment.

Reply:
Is it required though? Strangely we don't have that prop included today in `master` and it's not causing any issues. I remember seeing something like this the last time I touched this code.

Reply:
it only logs a warning in console, doesn't break typescript

Reply:
oh ok - I feel that we should ask if this parameter can be made optional, there's a definite use case for not needing to include it